src/test/java/com/indeed/jiraactions/TSVSpecBuilderTest.java

```diff
@@ -119,11 +119,11 @@ public void testUserCustomField() {
                 .multiValueFieldConfiguration(MultiValueFieldConfiguration.USERNAME)
                 .build();
         builder.addCustomFieldColumns(fieldDefinition);
-        final CustomFieldValue value = newCustomFieldValue(ImmutableList.of("user", "username"));
+        final CustomFieldValue value = newCustomFieldValue(ImmutableList.of("user", "username", "usergroups*|"));
```

Comment:
This test value is "wrong". I mean, it's not wrong per se, in that it's technically valid, but "*|" is a signifier to the TSV processor to tokenize this field on the '|' value. So putting it here is misleading.